src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -865,17 +865,23 @@ public class CaseData extends Et1CaseData {
     private Address et3RespondentAddress;
     @JsonProperty("et3ResponseDXAddress")
     private String et3ResponseDXAddress;
-    // ET3 Response - Phone number page (6)
-    @JsonProperty("et3ResponsePhone")
-    private String et3ResponsePhone;
-    // ET3 Response - What is your reference number (7)
-    @JsonProperty("et3ResponseReference")
-    private String et3ResponseReference;
-    // ET3 Response - Contact method page (8)
+    // ET3 Response - Representative Contact method, page (6)
     @JsonProperty("et3ResponseContactPreference")
     private String et3ResponseContactPreference;
     @JsonProperty("et3ResponseContactReason")
     private String et3ResponseContactReason;
+    // ET3 Response - Representative Contact language, page (6)
+    @JsonProperty("et3ResponseContactLanguage")
+    private String et3ResponseContactLanguage;
+    // ET3 Response - Representative Phone number page (6)
+    @JsonProperty("et3ResponsePhone")
```

Comment:
```suggestion
    @JsonProperty("et3ResponsePhoneNumber")
```

Reply:
@btensay it was implemented like that also in [master branch](https://github.com/hmcts/et-data-model/blob/master/src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java)(not my update). That field may have been used in another project.